src/platform/nrfconnect/DiagnosticDataProviderImplNrf.cpp

```diff
@@ -47,7 +47,10 @@ CHIP_ERROR DiagnosticDataProviderImplNrf::GetWiFiBssId(MutableByteSpan & value)
     WiFiManager::WiFiInfo info;
     ReturnErrorOnFailure(WiFiManager::Instance().GetWiFiInfo(info));
 
-    return CopySpanToMutableSpan(info.mBssId, value);
+    memcpy(value.data(), info.mBssId, sizeof(info.mBssId));
```

Comment:
Probably needs to check that `value.size()` is big enough.  `CopySpanToMutableSpan` does that automatically....

Reply:
Done